### DISCUSSION TITLE
SolidStart with-drizzle

### DISCUSSION THREAD

**Discussion by thednp** [solidjs/solid-start#1190] [Showcase]
Hi guys!

I've put some work into creating a `with-drizzle` example, similar to `with-prisma`. That is because I found no example anywhere, and drizzle is actually awesome!

Some notes:
* **drizzle-kit requires Node ^20 for some of its features**
* this example is set up with `better-sqlite3` and a local db file located in *./drizzle/db.sqlite*
* after making modifications to the schema, UI and actions, don't forget to push the migration to the db: `npx drizzle-kit push:sqlite`
* please test and report any problem

[with-drizzle-rc1.zip](https://github.com/solidjs/solid-start/files/15285563/with-drizzle-rc1.zip)

@ryansolid please feel free to include this into the official SolidStart examples. I can provide help in debugging it since I'm actively developing with it.

:smile:  Merry Christmas and Happy New Year!! :smile:

**Comment by thednp** [comment]
I've updated the dependencies and code.

**Comment by thednp** [comment]
Updated to SolidStart 0.5.0.

**Comment by thednp** [comment]
Updated to RC 1

**Comment by ryansolid** [comment]
I'm glad this works well. Thanks for letting us know. I haven't used Drizzle but if it is similar to Prisma definitely consider having an example for it.

**Comment by thednp** [comment]
Updated dependencies and some minor fixes.

**Comment by devagrawal09** [comment]
Any chance you can create a pull request adding that example into the repo?

**Comment by thednp** [comment]
Sure thing, I must update deps first.

I'm wondering @ryansolid should we improve the examples a bit? Like for instance have some minimal CSS or like some README specific to each example? It took me quite some time to figure out everything and would definitely come handy to new comers.

@devagrawal09 done
